Resources/Prototypes/Entities/Objects/Specific/Janitorial/trashbag.yml

```diff
@@ -11,7 +11,14 @@
   - type: Storage
     maxItemSize: Small
     grid:
-    - 0,0,7,5
+    - 3,0,5,0
+    - 3,1,3,1
+    - 5,1,5,1
+    - 0,2,8,2
+    - 1,3,2,9
+    - 6,3,7,9
+    - 4,3,4,6
+    - 1,7,7,9
```

Comment:
i don't really see the purpose in simultaneously buffing trashbags and making them awkward visually. the idea is fun but trashbags are too common of an item IMO to be getting this treatment. I'd rather just keep them as is as to not hamper their usability.

Reply:
Do people open trashbags regularly? When I played jani before making this I honestly thought the trash bags were disposable. It seems like the trashbag life cycle is click on ground to use auto fill -> click on disposal to empty with no reason i can think of to rummage around in it really.

Reply:
> Do people open trashbags regularly?

I sure do! Not all locations have disposal units right around the corner, and sometimes I wanna see how full it is before venturing into a trail of bullet casings. Granted, judging fullness was easier before gridinv, but even so. Plus, sometimes you wanna take things out that you didn't mean to hoover up.

This wonky interior shape really does feel needlessly gimmicky. It's literally just a bag, why does it need a silly shape?